jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -193,22 +193,34 @@ def __init__(self, *args, **kwargs):
             "flux_viewer_reference_name", "flux-viewer"
         )
 
+    def _is_spatial_subset(self, layer):
+        # spatial subset layers will have the same data-label as the collapsed flux cube
+        ref_data_label = self.layers[0].layer.data.label
```

Comment:
Is this more accurate? 🤷 
```suggestion
        ref_data_label = self.state.reference_data.label
```

Reply:
probably, I couldn't remember if there was an official way to access that.  Is there any other place that the "cube used for collapsing for the spectrum-viewer" is defined?

Reply:
I'll defer to other devs. The way it auto-collapses is a bit of a mystery to me still.